assets/monitoring/prometheus/v1/prometheusrule.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name:  "{{ .scyllaDBMonitoringName }}"
+  labels:
+    scylladbmonitoring-name: "{{ .scyllaDBMonitoringName }}"
+spec:
+  groups:
+  - name: ./example.rules
```

Comment:
we are not setting any yet, but this can be extended in the future

> shouldn't these be synced with what the dashboard is using?

is there a simple enough copy paste to use?

Reply:
> we are not setting any yet, but this can be extended in the future

IIRC dashboards are using some of the rules from https://github.com/scylladb/scylla-monitoring/tree/master/prometheus/prom_rules. I was setting it up for our docs not that long ago and without them our dashboards were a bit empty.

> is there a simple enough copy paste to use?

You could try what we are using currently (https://operator.docs.scylladb.com/v1.8/monitoring.html): `cd prometheus/prom_rules && find * -maxdepth 1 -type f -iregex '.*\.\(yaml\|yml\)' -print0 | xargs -0 yq ea '{(filename | sub("\/", "-")): .} | . as $item ireduce ({}; . * $item) | {"additionalPrometheusRulesMap": .}'` - probably needs some tweaking, this was used for ConfigMaps.

Reply:
I ran it with said rules just to make sure:
![image](https://user-images.githubusercontent.com/32443348/212682753-8de01e27-4885-448a-a106-be47bf467a7f.png)

Bear in mind it needs https://github.com/scylladb/scylla-monitoring/pull/1767 to parse.